### ISSUE TITLE
"org.bukkit.Bukkit.server" is null on 1.21

### ISSUE THREAD

**Issue by uprial** [MockBukkit/MockBukkit#1032]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the feature

When I use the mock on 1.21 I see:

```
	at be.seeseemelk.mockbukkit.MockBukkit.mock(MockBukkit.java:59)
...
Caused by: java.lang.NullPointerException: Cannot invoke "org.bukkit.Server.getRegistry(java.lang.Class)" because "org.bukkit.Bukkit.server" is null
	at org.bukkit.Bukkit.getRegistry(Bukkit.java:2140)
	at org.bukkit.Registry.<clinit>(Registry.java:105)
```

MockBukkit.java:59:
```
	public static @NotNull ServerMock mock()
	{
		return mock(new ServerMock());
	}
```
Could you please fix it?

**Comment by thelooter** [comment]
We don't support 1.21 yet, therefore this might happen. We're actively working on the support, but at this time, we can't provide this yet

**Comment by thelooter** [comment]
Can you check if this is covered by #1038?
Otherwise we will try to get a fix in before merging that PR

**Comment by Thorinwasher** [comment]
I will close this issue as fixed. 1.21 update has been merged now #1038
